scripts/update-component-sha.sh

```diff
@@ -1,10 +1,66 @@
 #!/bin/sh
 set -e
-if [ $# -ne 2 ] ; then
-    echo "Need <OLD> and <NEW> as arguments" >&2
+
+##
+#
+# script to replace hashes in config files
```

Comment:
Eh, let's leave it. I like having a short comment at the beginning of a script that says "this is what I do."